ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.14__amd64-nightly.yaml

```diff
@@ -503,29 +506,30 @@ tests:
     test:
     - chain: openshift-e2e-test-qe-destructive
     workflow: cucushift-installer-rehearse-aws-ipi-disconnected-private
-- as: aws-ipi-disconnected-sdn-rhcos-rhel8-p2-f7
-  cron: 25 20 5,13,20,27 * *
+- as: aws-ipi-disconnected-sdn-rhcos-rhel8-fips-p2-f7
```

Comment:
hmm, this workflow does not support fips yet, more details are mentioned in https://docs.google.com/spreadsheets/d/1mgPxhb4iUsPQfItdj9rzHGL0-iLY6GNFs5pijIW7G6o/edit#gid=492690660.